logic-and-algorithm-test/2.py

```diff
@@ -1,18 +1,18 @@
 # Diberikan sebuah string acak, hitunglah berapa jumlah setiap karakter yang ada dalam string
 # tersebut
 def count_characters(input_string):
-    char_count = {}
+    character_count = {}
     for char in input_string:
-        if char in char_count:
-            char_count[char] += 1
+        if char in character_count:
+            character_count[char] += 1
         else:
-            char_count[char] = 1
-    return char_count
+            character_count[char] = 1
```

Comment:
Ensure consistent renaming throughout the code to avoid reference errors. Good job on maintaining consistency here.